lib/beaker/host/unix/pkg.rb

```diff
@@ -514,7 +514,7 @@ def install_local_package(onhost_package_file, onhost_copy_dir = nil)
       command_name = 'dnf 'if variant == 'fedora' && version > 21 && version <= 29
       execute("#{command_name} --nogpgcheck localinstall -y #{onhost_package_file}")
     when /^(sles)$/
-      execute("rpm -ihv #{onhost_package_file}")
+      execute("zypper --non-interactive --no-gpg-checks in #{onhost_package_file}")
```

Comment:
So all the versions of SLES we support have zypper, correct?

Reply:
Correct.